app/src/main/java/com/example/tmdb/screens/home/HomeViewModel.kt

```diff
@@ -0,0 +1,203 @@
+package com.example.tmdb.screens.home
+
+import androidx.compose.runtime.State
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import androidx.paging.PagingData
+import androidx.paging.cachedIn
+import androidx.paging.filter
+import com.example.tmdb.model.*
+import com.example.tmdb.repository.MoviesRepository
+import com.example.tmdb.repository.TvRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.emptyFlow
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class HomeViewModel @Inject constructor(
+    private val moviesRepository: MoviesRepository,
+    private val tvRepository: TvRepository,
+) : ViewModel(){
+
+    // Movie
+    private var _trendingMoviesDay = mutableStateOf <Flow<PagingData<Movie>>>(emptyFlow())
+    val trendingMoviesDay: State<Flow<PagingData<Movie>>> = _trendingMoviesDay
```

Comment:
No need to wrap this into State. 

Expose Flows from your ViweModel and then you can convert it into state in your Views.